devito/symbolics/manipulation.py

```diff
@@ -334,6 +334,9 @@ def evalrel(func=min, input=None, assumptions=None):
     assumptions : list
         A list of assumptions formed as relationals between candidates, assumed to be
         True. Defaults to None.
+    eval : bool, optional
```

Comment:
maybe have two functions, `reduce_relation` and `eval_relation`, where the latter essentially calls the former plus one extra line for evaluation?

I don't like much two different output types here

Reply:
DOne